README.md

```diff
@@ -219,6 +219,47 @@ let () = printGithubRepos ()
 ## Node-specific
 
 #### Read lines from a text file
+
+```ml
+let () =
+    Node_fs.readFileSync "README.md" `utf8
```

Comment:
`bs-node` is a more up to date package that will eventually replace the Node module that ships with bs-platform. It also has a `readFileAsUtf8Sync` function